packages/gatsby-transformer-xml/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -4,12 +4,8 @@ exports[`Process XML nodes correctly correctly creates nodes from XML 1`] = `
 Array [
   Array [
     Object {
-      "attributes": Object {
-        "id": "bk101",
-      },
```

Comment:
`attributes` seems to no longer show up like this - instead they go into `xmlChildren`:

```
      "xmlChildren": Array [
        Object {
          "content": "bk101",
          "name": "attributes",
        },
```

Reply:
This should be resolved with the latest commits.